app/assets/javascripts/initializers/initializeReadingListIcons.js

```diff
@@ -14,7 +12,9 @@ function initializeReadingListIcons() {
 
 // set SAVE or SAVED articles buttons
 function setReadingListButtonsState() {
-  var readingListButtons = document.getElementsByClassName('bookmark-button');
+  var readingListButtons = document.querySelectorAll(
+    '.bookmark-button:not([data-initial-feed])',
```

Comment:
I added `:not([data-initial-feed]` which is on the `<SaveButton />` component so that this only picks up the reading list buttons from Algolia paging.